cclib/method/ddec.py

```diff
@@ -33,7 +33,7 @@ class DDEC6(Method):
     required_attrs = ("homos", "mocoeffs", "nbasis", "gbasis")
 
     def __init__(
-        self, data, volume, proatom_path=None, progress=None, loglevel=logging.INFO, logname="Log"
+        self, data, volume, proatom_path=None, progress=None, convergence_level = 1e-10, loglevel=logging.INFO, logname="Log"
     ):
         # Inputs are:
```

Comment:
Could you put this into a docstring?

Reply:
I've made this into a docstring. Thanks.